README.md

```diff
@@ -103,16 +101,15 @@ Email keystone.engine@gmail.com for any questions.
 
 For future update of Keypatch, follow our Twitter [@keystone_engine](https://twitter.com/keystone_engine) for announcement.
 
-
 ----
 
 ### Appendix. Install Keystone for IDA Pro
 
-We all know that before IDA 7.0, IDA Pro's Python is 32-bit itself, so it can only loads 32-bit libraries. For this reason, we have to build & install Keystone 32-bit. However, IDA 7.0 is now a native 64-bit application, which means we also need to install a correct version of Keystone. This section details the steps towards that goal.
+We all know that <= IDA 7.0 (eg: 6.8, 6.9, 6.95, 7.0_x86), IDA Pro's Python is 32-bit itself, so it can only loads 32-bit libraries. For this reason, we have to build & install Keystone 32-bit. However, IDA 7.0 is now a native 64-bit application, which means we also need to install a correct version of Keystone. This section details the steps towards that goal.
```

Comment:
why did you change "before" to "<=", which indicates also v7.0 (due to `=`) here?

Reply:
IDA7.0 contains (32bit, 64bit). @aquynh

Reply:
yes, but that is confused. how about keeping that "before" part, but change the later part "IDA 7.0 is now a native 64-bit application ..." to "since IDA 7.0 supports both 32-bit & 64-bit ..." ?

Reply:
@aquynh modified